scripts/extractStrings.js

```diff
@@ -15,7 +15,8 @@ const LocKeys = [
   "defaultItemsText",
   "loadButtonText",
   "noDataMessage",
-  "markDown" // specific to cohorts
+  "markDown", // specific to cohorts
+  "actionName"
```

Comment:
>actionName [](http://example.com/codeflow?start=3&length=10)

why actionname?  isn't this `runLabel` and `actionMessage` and `resultMessage` and things?

Reply:
should be actionName and runLabel